src/docs/product/relay/operating-guidelines.mdx

```diff
@@ -113,3 +114,72 @@ limits:
 ```
 
 See the [Configuration Options](/product/relay/options/) page for detailed descriptions of all available options.
+
+## Performance Tuning
+
+Relay has quite a lot of different [Configuration Options](/product/relay/options/), and
+changing some of them can have more impact on Relay's behavior than others.
+
+Here we list a few relevant options that can help you to tune Relay to specific environment and workload:
```

Comment:
```suggestion
Relay provides a variety of [Configuration Options](/product/relay/options/). Changing some of the options has more impact on Relay's behavior than others. The following list identifies options that help you tune Relay to your organization's environment and workload:

Reply:
We would like to emphasize that this is just a small subset of the options, but you should start with them if you want to tweak Relay's performance and buffering behavior.

What do you think about this:
```suggestion
Relay provides a variety of [Configuration Options](/product/relay/options/). Changing some of the options has more impact on Relay's behavior than others. 
The following list identifies a few options you should check first when you want to tune Relay to your organization's environment and workload:
```

Reply:
Yes, I think this works very well.